code/addons/essentials/src/measure/preview.ts

```diff
@@ -1,2 +1 @@
-// @ts-expect-error (no types needed for this)
```

Comment:
I have no idea why this `addon-measure` module in `addon-essentials` is the only one where the `@ts-expect-error` needs to be removed. None of the other (seemingly identical) modules needs this removed.

@ndelangen ?

Reply:
It's because of this:
https://github.com/storybookjs/storybook/blob/bd7ce6f5f83c1cbb6bf32b841ba705023ad416e8/code/addons/measure/package.json#L46-L58